metadata-ingestion/src/datahub/ingestion/source/aws/glue.py

```diff
@@ -304,7 +304,9 @@ def get_all_jobs(self):
 
         return jobs
 
-    def get_dataflow_graph(self, script_path: str) -> Optional[Dict[str, Any]]:
+    def get_dataflow_graph(
+        self, script_path: str, flow_urn: str
+    ) -> Optional[Dict[str, Any]]:
         """
```

Comment:
Tracking is easier with key flow_urn. script_path is already present in reported message.